extensions/binding/org.eclipse.smarthome.binding.lifx/src/main/java/org/eclipse/smarthome/binding/lifx/internal/protocol/GetColorZonesRequest.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.lifx.internal.protocol;
+
+import static org.eclipse.smarthome.binding.lifx.LifxBindingConstants.*;
+
+import java.nio.ByteBuffer;
+
+import org.eclipse.smarthome.binding.lifx.internal.fields.Field;
+import org.eclipse.smarthome.binding.lifx.internal.fields.UInt8Field;
+
+/**
+ * @author Wouter Born - Add support for MultiZone light control
+ */
+public class GetColorZonesRequest extends Packet {
+
+    public static final int TYPE = 0x1F6;
+
+    public static final Field<Integer> FIELD_START_INDEX = new UInt8Field();
+    public static final Field<Integer> FIELD_END_INDEX = new UInt8Field();
+
+    private int startIndex = MIN_ZONE_INDEX;
+    private int endIndex = MAX_ZONE_INDEX;
+
+    public GetColorZonesRequest() {
+        setTagged(false);
+        setAddressable(true);
+        setResponseRequired(true);
+    }
+
+    public GetColorZonesRequest(int index) {
+        this(index, index);
+    }
+
+    public GetColorZonesRequest(int startIndex, int endIndex) {
+        this();
```

Comment:
I think that the methods in `GetColorZonesRequest()` would no longer be called? That would result in `protocol = 0` in the parent `Packet` class. Especially the `addressable` protocol bit needs to be set in most packets. See the LIFX protocol [header description](https://lan.developer.lifx.com/docs/header-description).

A lot of the LIFX protocol objects have the same protocol flags being set in their default constructor. This means that the current default value is not properly chosen. I think a good default would be:
* tagged = false (only set to true for discovery packets)
* addressable = true (most messages are sent to a single light)
* responseRequired = false (these response packets do not contain current values, already set to false in `LifxLightStateChanger.createPendingPackets(Packet...)`

Because changing these defaults impacts all packets I would rather do that in another PR after more testing.